R/pkg/R/mllib_regression.R

```diff
@@ -109,7 +125,8 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
             # For known families, Gamma is upper-cased
             jobj <- callJStatic("org.apache.spark.ml.r.GeneralizedLinearRegressionWrapper",
                                 "fit", formula, data@sdf, tolower(family$family), family$link,
-                                tol, as.integer(maxIter), as.character(weightCol), regParam)
+                                tol, as.integer(maxIter), as.character(weightCol), regParam,
+                                as.double(variancePower), as.double(linkPower))
```

Comment:
we probably don't need to `as.double` here since it is either set to fixed values (L116) or from a calculation. Instead, we should check `var.power` and `link.power` are within the correct range - not sure if the tweedie function does that.